src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -110,6 +110,14 @@ export class CalcitePickListItem {
     this.emitChangeEvent(event.shiftKey);
   };
 
+  pickListKeyDownHandler = (event: KeyboardEvent): void => {
+    if (event.key === " ") {
```

Comment:
Should we support the enter key too?

Reply:
We could. Usually that submits a form when working with generic components, so I erred on keeping things consistent. Still leaning that way.